src/integrations/contact-input-custom-chip-component.tsx

```diff
@@ -278,10 +283,20 @@ const CustomComponent = (props: CustomChipProps): JSX.Element => {
 };
 
 export const ContactInputCustomChipComponent = (props: CustomChipProps): ReactElement => {
-	const { email, isGroup } = props;
+	const { email, isGroup, label, chipDisplayName = CHIP_DISPLAY_NAME_VALUES.LABEL } = props;
```

Comment:
By extracting the props here instead of inside the previus line, all the props that are not part of the Chip definition are still passed to the component. I feel it a bit dangerous, especially for those kind of props that match valid html attributes, which will ends up being passed down to the final html.

Reply:
I did some changes, let me know if now is better